Dockerfile

```
@@ -1,5 +1,5 @@
 # Build node-purple. We need debian for python3.6, which is needed for node-purple
-FROM node:20-bookworm as builder
+FROM node:22-bookworm as builder
```

Nitpick, to avoid a warning from `docker build`:
```suggestion
FROM node:22-bookworm AS builder
```